skymp5-functions-lib/src/types/mp.d.ts

```diff
@@ -126,7 +126,7 @@ export interface Mp {
   set(formId: number, propertyName: 'locationalData', newValue: LocationalData): void;
   set(formId: number, propertyName: 'spawnPoint', newValue: LocationalData): void;
   set(formId: number, propertyName: 'isDead', newValue: boolean): void;
-  set(formId: number, propertyName: 'percentages', newValue: { health: number, magicka: number, stamina: number }): void;
+  set(formId: number, propertyName: 'percentages', newValue: { health: number; magicka: number; stamina: number } ): void;
```

Comment:
Formatting changes are still here. We don't need them until we will reformat all scripts with some codestyle.